packages/x-adapter/README.md

```diff
@@ -409,22 +409,221 @@ export const searchUsersWithContactInfo = endpointAdapterFactory({
+  postId: 'postId'
+});
+
+const commentsResponse = schemaMapperFactory<ApiCommentsResponse, AppCommentsResponse>({
+  comments: {
+    $subSchema: commentSchema,
+    $path: 'comments'
+  }
+});
+
+export const searchComments = endpointAdapterFactory({
+  endpoint: 'https://dummyjson.com/comments',
+  responseMapper: commentsResponse
+});
+```
+
+###### Override a MutableSchema to add more fields
+
+As said above, the suitable context for using the `override` method would say a project with an API
+that doesn't differ too much against the one used in our "base project". That means we can reuse
+most of the types and schemas definitions, so we would only add a few new fields from the new API.
+
+```ts
+import { ApiBaseObject, AppBaseObject, baseObjectSchema } from '@/base-types';
+
+// Api models
+interface ApiTodo {
+  completed: boolean;
+  todo: string;
+  userId: number;
+}
+
+interface ApiTodosResponse {
+  todos: ApiBaseObject[];
+}
+
+// App models
+interface AppTodo {
+  completed: boolean;
+  text: string;
+  userId: string;
+}
+
+interface AppTodosResponse {
+  todos: AppBaseObject[];
+}
+
+// Override the original Schema. The Schema changes to map: 'id', 'completed', 'text' and 'userId''
+baseObjectSchema.$override<ApiTodo, AppTodo>({
+  completed: 'completed',
+  text: 'todo',
+  userId: ({ userId }) => userId.toString()
+});
+
+// Response mapper
+const todosResponse = schemaMapperFactory<ApiTodosResponse, AppTodosResponse>({
+  todos: {
+    $subSchema: baseObjectSchema,
+    $path: 'todos'
+  }
+});
+
+// Endpoint Adapter
+export const searchTodos = endpointAdapterFactory({
+  endpoint: 'https://dummyjson.com/todos',
+  responseMapper: todosResponse
+});
+```
+
+###### Replace a MutableSchema to completely change it
+
+In this case we are facing too many differences between API responses. We don't need to write a
+whole adapter from scratch, as there are other parts of the API that aren't changing so much, but we
+should replace some `endpointAdapter`'s schemas.
+
+```ts
+import { ApiBaseObject, AppBaseObject, baseObjectSchema } from '@/base-types';
+
+// Api models
+interface ApiQuote {
+  id: number;
+  quote: string;
+  author: string;
+}
+
+interface ApiQuotesResponse {
+  quotes: ApiBaseObject[];
+}
+
+// App models
+interface AppQuote {
+  quoteId: string;
+  quote: string;
+  author: string;
+}
+
+interface AppQuotesResponse {
+  quotes: AppBaseObject[];
+}
+
+// Replace the original Schema
+baseObjectSchema.$replace<ApiQuote, AppQuote>({
+  quoteId: ({ id }) => id.toString(),
+  quote: 'quote',
+  author: 'author'
+});
+
+// Response mapper
+const quotesResponse = schemaMapperFactory<ApiQuotesResponse, AppQuotesResponse>({
+  quotes: {
+    $subSchema: baseObjectSchema,
+    $path: 'quotes'
+  }
+});
+
+// Endpoint Adapter
+export const searchQuotes = endpointAdapterFactory({
+  endpoint: 'https://dummyjson.com/quotes',
+  responseMapper: quotesResponse
+});
```

Comment:
To emphasize that `$replace` and `$override` mutate the original schemas, and therefor the adapter, I would move the `$replace` and `$override` calls after the creation of the endpoint adapter.